proofs-plugin/src/main/java/arrow/meta/diagnostic/MetaDefaultErrorMessages.java

```diff
@@ -27,13 +25,12 @@ public DiagnosticFactoryToRendererMap getMap() {
         );
         MAP.put(AmbiguousProof,
                 "This {0} has following conflicting proof/s: {1}.\nPlease disambiguate resolution, by either declaring only one internal orphan / public proof over the desired type/s or remove conflicting proofs from the project.", RenderProof, RenderProofs);
+        MAP.put(AmbiguousProofForSupertype,
+                "Found ambiguous proofs for type {0}. Proofs : {2}", RenderTypes, RenderProof, RenderProofs);
         MAP.put(OwnershipViolatedProof,
                 "This {0} violates ownership rules, because public Proofs over 3rd party Types break coherence over the kotlin ecosystem. One way to solve this is to declare the Proof as an internal orphan.", RenderProof);
         MAP.put(UnresolvedGivenProof, "This GivenProof on the type {0} can't be semi-inductively resolved. Please verify that all parameters have default value or that other injected given values have a corresponding proof.", RenderTypes);
+        MAP.put(CycleOnGivenProof, "This GivenProof on the type {0} has cyclic dependencies: {1}. Please verify that proofs don't depend on each other for resolution.", RenderTypes, RenderProofs);
```

Comment:
These messages are not autoshown in the IDEA, are they?

Reply:
I believe they will not show up immediately as other analysis errors do but will be reported as errors with the right code location when a full build is performed. Same as codegen and other plugins that have no IDEA support yet until FIR compiler plugins are a thing.